<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>isDogu</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1261</width>
      <height>371</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>현재 모니터링 상태</string>
     </attribute>
     <widget class="QGroupBox" name="description_captured_packets">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>411</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>맑은 고딕</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>수집된 패킷</string>
      </property>
      <widget class="QLabel" name="captured_packets">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>맑은 고딕</family>
         <pointsize>20</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>10</y>
        <width>801</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>맑은 고딕</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>선택된 네트워크 인터페이스</string>
      </property>
      <widget class="QLabel" name="selected_network">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>35</y>
         <width>761</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>맑은 고딕</family>
         <pointsize>20</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="description_kdd">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>611</width>
        <height>251</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>맑은 고딕</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>NSL-KDD</string>
      </property>
      <widget class="QLabel" name="kdd_data_total">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>60</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>맑은 고딕</family>
         <pointsize>20</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="kdd_probe_warning">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>180</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>맑은 고딕</family>
         <pointsize>20</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>맑은 고딕</family>
         <pointsize>20</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>분석된 데이터 수</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>321</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>맑은 고딕</family>
         <pointsize>20</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>DoS로 탐지된 공격</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>301</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>맑은 고딕</family>
         <pointsize>20</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Probe로 탐지된 공격</string>
       </property>
      </widget>
      <widget class="QLabel" name="kdd_dos_warning">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>120</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>맑은 고딕</family>
         <pointsize>20</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="description_cic">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>90</y>
        <width>611</width>
        <height>251</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>맑은 고딕</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>CSE-CIC-IDS 2018</string>
      </property>
      <widget class="QLabel" name="cic_data_total">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>60</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>맑은 고딕</family>
         <pointsize>20</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="cic_bf_warning">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>120</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>맑은 고딕</family>
         <pointsize>20</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>311</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>맑은 고딕</family>
         <pointsize>20</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>BruteForce로 탐지된 공격</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>맑은 고딕</family>
         <pointsize>20</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>분석된 데이터 수</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>311</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>맑은 고딕</family>
         <pointsize>20</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>DDoS로 탐지된 공격</string>
       </property>
      </widget>
      <widget class="QLabel" name="cic_ddos_warning">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>180</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>맑은 고딕</family>
         <pointsize>20</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>로그 정보</string>
     </attribute>
     <widget class="QTextBrowser" name="log_box">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>50</y>
        <width>781</width>
        <height>281</height>
       </rect>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
     </widget>
     <widget class="QTextBrowser" name="ip_log_box">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>441</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>맑은 고딕</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>수신한 패킷 기본 정보</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>10</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>맑은 고딕</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>탐지된 공격 정보</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>1261</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="graph_layout"/>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>5</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>NIDS with A.I. - Made by Team Dogu</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>프로그램</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>정보</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>종료</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
